notes/git-managing-branches.md

```diff
@@ -0,0 +1,81 @@
+**Managing branches**
```

Comment:
It is also to be consistent when u write documentation. 
## MANAGING ...
check the other file

Reply:
The header is now in bold text, thanks for spotting this.

Reply:
But again, it is not consistent, your headings are in CAPITAL LETTERS as in the comment before. Why should be this one in lowercase?

Reply:
True, I focused on the Markdown. Fixed now.